src/throw-from-number.js

```diff
@@ -0,0 +1,18 @@
+function getThrowFromNumber(number) {
+
+    if(number === 0) {
+        return 'rock';
+    }
+    
+    else if(number === 1) {
+        return 'paper';
+    }
+    
+    else if(number === 2) {
+        return 'scissors';
+    }
+     
+    return; 
```

Comment:
You don't need this return - since you're only going to get 0, 1, 0r 2 as an input, every case has been handled and you don't need a final return. If you wanted to have a catch for an error, like '3' being unexpectedly entered, you could do it as a final "else" block. If that doesn't make sense, feel free to grab me!